File: accelerators/aks-sb-azmonitor-microservices/src/valid-cargo-manager/Models/OperationState.cs
namespace ValidCargoProcessor
{
    using Newtonsoft.Json;
    public class OperationState
    {
        [JsonProperty(PropertyName = "operationId")]
        public string? OperationId { get; set; }
        [JsonProperty(PropertyName = "state")]
        public string? State { get; set; }
        [JsonProperty(PropertyName = "result")]
        public ValidCargo? Result { get; set; }
        [JsonProperty(PropertyName = "error")]
        public string? Error { get; set; }
    }
}

File: accelerators/aks-sb-azmonitor-microservices/src/operations-api/src/main/java/com/microsoft/cse/cargoprocessing/operations/api/models/OperationState.java
package com.microsoft.cse.cargoprocessing.operations.api.models;

import java.io.Serializable;

import lombok.Data;

@Data
public class OperationState implements Serializable {
    private String operationId;
    private String state;
    private Cargo result;
    private String error;
}
